src/components/media/Media.scss

```diff
@@ -27,7 +27,8 @@
 }
 
 // the "..." menu button, meant to be used inside Media-body
-.bdl-Media-menu {
+.bdl-Media-menu,
+button.bdl-Media-menu {
```

Comment:
- & doesn't seem to parse unless you are in the `{}` block
- could put `.btn-plain` inside but we might have a different class soon